source/Core/Src/Settings.cpp

```diff
@@ -370,3 +374,6 @@ uint8_t getUserSelectedTipResistance() {
     break;
   }
 }
+#else
+uint8_t getUserSelectedTipResistance() { return tipType_t::TIP_TYPE_AUTO; }
```

Comment:
Instead of twisting my head about how to do it inside of the existed instance of `getUserSelectedTipResistance()`, I thought that the most simple & straight-forward approach is just to make the duplicated stub with the single return.